remote/src/main/scala/org/apache/pekko/remote/PhiAccrualFailureDetector.scala

```diff
@@ -119,7 +117,8 @@ class PhiAccrualFailureDetector(
   private val acceptableHeartbeatPauseMillis = acceptableHeartbeatPause.toMillis
 
   // address below was introduced as a var because of binary compatibility constraints
```

Comment:
Can we prefix this comment with `NOTE` so that its easily searchable. Should also make a pekko issue to make this a non var targeting Pekko 2.0.x